test/helpers/cons.go

```diff
@@ -187,9 +187,9 @@ const (
 
 	// CiliumStableHelmChartVersion should be the chart version that points
 	// to the v1.X branch
-	CiliumStableHelmChartVersion = "1.9-dev"
-	CiliumStableVersion          = "v1.9"
-	CiliumLatestHelmChartVersion = "1.9.90"
+	CiliumStableHelmChartVersion = "1.10-dev"
+	CiliumStableVersion          = "v1.10"
+	CiliumLatestHelmChartVersion = "1.10.90"
```

Comment:
These changes should not be backported to v1.10, as it represents the target version to start upgrade tests from.

Reply:
Hmm right, I went back and forth in my head whether to create to separate PRs, but then thought for some reason that both commits need to be backported. Good catch. I'll handle the backport myself and pull it manually into https://github.com/cilium/cilium/pull/15919.